test/performance/observed_concurency_test.go

```diff
@@ -136,7 +145,19 @@ func TestObservedConcurrency(t *testing.T) {
 	test.CleanupOnInterrupt(func() { TearDown(perfClients, names, t.Logf) })
 
 	t.Log("Creating a new Service")
-	objs, err := test.CreateRunLatestServiceReady(t, clients, &names, &test.Options{ContainerConcurrency: 1})
+	objs, err := test.CreateRunLatestServiceReady(t, clients, &names, &test.Options{
+		ContainerConcurrency: 1,
+		ContainerResources: corev1.ResourceRequirements{
+			Limits: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("10m"),
+				corev1.ResourceMemory: resource.MustParse("50Mi"),
+			},
+			Requests: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("10m"),
+				corev1.ResourceMemory: resource.MustParse("20Mi"),
+			},
+		},
```

Comment:
I assume this is tuning down on the resource requirements of the app. Should that be done in the test or via config now that https://github.com/knative/serving/commit/306de3d6b7f2323e65bed38d6542c28e8bce642a has landed?

Reply:
Yes. This is to cut down on resource requirements so as to allow for large-scale tests. I think the configuration should be done in the test, similar to all the other tests that manipulate the resource requirements (none of them use/change the new config map). We want to configure our own test and not default values affecting everyone else. Thanks for pointing out these new changes though.

Reply:
Would it suffice though to only specify the `Requests` section of it? Specifying such narrow limits seems a bit of an overkill to me.

Reply:
OK. I've tested several times against a remote cluster and was reliably getting good responses but I guess it wouldn't hurt to get rid of the limit.